priv/rest_json.ex.eex

```diff
@@ -68,6 +68,7 @@ defmodule <%= context.module_name %> do
       {:ok, response=%HTTPoison.Response{status_code: 204, body: body}} ->
         {:ok, Poison.Parser.parse!(body), response}
       {:ok, _response=%HTTPoison.Response{body: body}} ->
+        :timer.sleep(50)
```

Comment:
This is just sleeping before returning a response, the HTTP request has already been made.

I think we might want to add a new case where we get an `:error` and the HTTPoison.Response status code is 429 ("Too Many Requests") where we retry the request?

Or we could just put a sleep before every request. May mean fewer HTTP requests but will definitely slow down every request.